lib/prefix.h

```diff
@@ -311,10 +311,17 @@ union prefixconstptr {
 #endif /* INET_ADDRSTRLEN */
 
 #ifndef INET6_ADDRSTRLEN
+/* Use this for GUA or LL IPv6 address len */
 /* dead:beef:dead:beef:dead:beef:dead:beef + \0 */
 #define INET6_ADDRSTRLEN 46
 #endif /* INET6_ADDRSTRLEN */
 
+#ifndef INET6_GUA_ADDRSTRLEN
+/* Use this for GUA IPv6 addresss len */
+/* dead:beef:dead:beef:dead:beef:dead:beef + \0 */
+#define INET6_GUA_ADDRSTRLEN 40
```

Comment:
don't follow this at all. the "STRLEN" macros are used to specify a buffer size for binary-to-ascii conversion, not for vty output formatting?
and ... typo?
and ... the example string is exactly the same as the existing macro?

Reply:
if it's for a column width, let's name it that way then?

Reply:
@mjstapp I moved it out of lib/prefix.h and it is a very useful when trying to print just GUA address.